frontend/src/components/receipts/ReceiptsList.tsx

```diff
@@ -5,18 +5,14 @@ import Placeholder from "../utils/Placeholder";
 import PaginationSpinner from "../utils/PaginationSpinner";
 
 import { useTranslation } from "react-i18next";
-import { useWampSimpleQuery } from "../../hooks/wamp";
+import { Receipt } from "../../libraries/wamp/types";
 
 interface Props {
-  blockHash: string;
+  receiptsList?: Receipt[];
```

Comment:
Why is that optional?

Reply:
Because on `ReceiptsList` component we show loader if `receiptsList: undefined` (on the page load) otherwise an array. Also `useWampSimpleQuery` returns `T | undefined` so in my opinion it's ok. Maybe it's better to return an empty array as an initial value but then the placeholder message will keep popping up and only after some time will be replaced by data